src/widgets/connector.ts

```diff
@@ -378,17 +378,45 @@ export class ConnectorView extends DOMWidgetView {
 
         const connectionTemplate = this.connectionsTemplates[key];
 
+        const form: { [key: string]: any } = {};
```

Comment:
please give this a more descriptive name, it's unclear what the purpose of this variable is

Reply:
[fixed](https://github.com/ploomber/jupysql-plugin/pull/75/files#diff-245ccc293a87304bf06e6d14a672d745b47a2cc1951a41def9f729b25ee669e1R381-R382)